test-utils/state-viewer/src/main.rs

```diff
@@ -37,7 +40,7 @@ fn load_trie(
     for chunk in last_block.chunks.iter() {
         state_roots.push(chunk.inner.prev_state_root.clone());
     }
-    (runtime, state_roots, last_block.header.inner_lite.height)
```

Comment:
above `let last_block = ..` should it actually be getting last _final_ block?

Reply:
I plan to do that in a separate PR as it will change how state dump works: it needs to take one argument as the "expected height" and we will dump state from the first final block with height >= expected height. This way we can guarantee that the state dump result is consistent on every node.

Reply:
+1